lantern.nsi

```diff
@@ -0,0 +1,86 @@
+Name "Lantern"
+
+# Installs Lantern and launchs it
+# See http://nsis.sourceforge.net/Run_an_application_shortcut_after_an_install
+
+AutoCloseWindow true
+
+!addplugindir nsis_plugins
+!include "nsis_includes/nsProcess.nsh"
+
+# Use the modern ui
+!include MUI.nsh
+!define MUI_ICON lantern.ico
+
+;Languages
+!insertmacro MUI_LANGUAGE "Farsi"
+!insertmacro MUI_LANGUAGE "English"
+!insertmacro MUI_LANGUAGE "Czech"
+!insertmacro MUI_LANGUAGE "Dutch"
+!insertmacro MUI_LANGUAGE "French"
+!insertmacro MUI_LANGUAGE "German"
+!insertmacro MUI_LANGUAGE "Korean"
+!insertmacro MUI_LANGUAGE "Russian"
+!insertmacro MUI_LANGUAGE "Spanish"
+!insertmacro MUI_LANGUAGE "Swedish"
+!insertmacro MUI_LANGUAGE "TradChinese"
+!insertmacro MUI_LANGUAGE "SimpChinese"
+!insertmacro MUI_LANGUAGE "Slovak"
+
+# define name of installer
+OutFile "lantern-installer-unsigned.exe"
+ 
+# define installation directory
+InstallDir $PROGRAMFILES\Lantern
+ 
+# For removing Start Menu shortcut in Windows 7
+RequestExecutionLevel admin
+    
+# start default section
+Section
+    # Stop existing Lantern if necessary
+    ${nsProcess::KillProcess} "lantern.exe" $R0
+    # Sleep for 1 second to give file lock a chance to clear
```

Comment:
Maybe not the best wording.  The file is locked to writers by the OS while the program is running.  After killing the program, there's a short delay for that lock to clear.  That's what this refers to.